packages/web/src/pages/upload-page/fields/availability/UsdcPurchaseFields.module.css

```diff
@@ -5,3 +5,7 @@
   border: 1px solid var(--border-strong);
   background: var(--background-surface-1);
 }
+
+.input > div:first-child {
```

Comment:
Is this against our css guidelines? The `InputV2` className prop is a level up from this. Any better ideas on how to make this happen?

Reply:
i feel like the background should be white by default in the component right? it was probably not set bc its normally used on a white background, but i can't thing of a case where we would want it to have a transparent background.

Reply:
alternatively, we could add a prop like inputClassName or something so that we can just pass the class to the right spot

Reply:
InputV2 has `background-color: var(--neutral-light-10);` and I didn't want to break that, but yeah could just pass another prop

Reply:
ooh thats weird. sounds good 👍

Reply:
done